site/content/api/Telemetry-API.md

```diff
@@ -65,6 +65,7 @@ The following table provides an overview of the properties a client needs to set
 | *content-type* | yes       | *properties*             | *symbol*  | SHOULD be set to *application/octet-stream* if the message payload is to be considered *opaque* binary data. In most cases though, the client should be able to set a more specific content type indicating the type and characteristics of the data contained in the payload, e.g. `text/plain; charset="utf-8"` for a text message or `application/json` etc. |
 | *device_id*    | yes       | *application-properties* | *string*  | MUST contain the ID of the device the data in the payload has been reported by. |
 | *reg_assertion*| yes       | *application-properties* | *string*  | A [JSON Web Token](https://jwt.io/introduction/) issued by the [Device Registration service]({{< relref "api/Device-Registration-API.md#assert-device-registration" >}}) asserting the device's registration status. |
+| *stay_connected* | no      | *application-properties* | *long*  | In context with the `creation-time` of the AMQP 1.0 message it defines the time interval in milliseconds in which a device should be available for receiving upstream data. If this property is missing, it shall be considered as not being known if the device is ready for receiving a message. If it has the value 0, the device shall be considered as being always ready to receive a message. | 
```

Comment:
again, I propose to rename to `ttd`

Instead of 0 I propose to  use -1 as the indicator for *always online* because it is completely out-of-reasonable-range.